src/D07/Bag.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.Generic;
+
+namespace aoc.D07
+{
+	public class Bag
+	{
+		private string _color;
+		private Dictionary<string, int> _table;
+
+		public Bag(string color)
+		{
+			_color = color;
+			_table = new Dictionary<string, int>();
+		}
+
+		public string Color { get { return _color; } }
+
+		public Dictionary<string, int> Table { get { return _table; } }
```

Comment:
I would rename this - Table doesn't make much sense

Reply:
renamed to Data